tool/tsh/tsh.go

```diff
@@ -908,32 +908,33 @@ func onSSH(cf *CLIConf) {
 	err = client.RetryWithRelogin(cf.Context, tc, func() error {
 		return tc.SSH(cf.Context, cf.RemoteCommand, cf.LocalExec)
 	})
-	if err != nil {
```

Comment:
Is it ok to make smaller refactorings as I stumble on things?
Used an earlier return to avoid unnecessary indentation.